### PR TITLE
Adds NewRelic support

### PR DESCRIPTION
**Pull request by nickschuch**
Adds NewRelic configuration with [environment variables](https://github.com/skpr/operator/blob/master/internal/controller/app/drupal/controller.go#L75)

### CODE REVIEW

File: php/base/newrelic.ini [hunk 1]
```diff
@@ -0,0 +1,858 @@
...
+; Type   : string
+; Scope  : per-directory
+; Default: "PHP Application"
+; Info   : Sets the name of the application that metrics will be reported into.
+;          This can in fact be a list of up to 3 application names, each of
+;          which must be separated by a semi-colon. The first name in any such
+;          list is considered the 'primary' application name and must be unique
+;          for each account / license key.
+;
+newrelic.appname = "${NEW_RELIC_APP_NAME}"
```
Comment (nicksantamaria) [inline php/base/newrelic.ini]: Is this environment variable evaluated at runtime?

Comment (nicksantamaria) [reply php/base/newrelic.ini]: We should document this envvar somewhere. Maybe skpr/example ?

Comment (nickschuch) [reply php/base/newrelic.ini]: https://github.com/skpr/example/commit/b413db157ed41ffafe55c976918a2eb0269af387

Comment (nickschuch) [reply php/base/newrelic.ini]: I have also pushed these as ENV items on the PHP Dockerfile

**Pull request merged**